charts/tidb-cluster/values.yaml

```diff
@@ -706,6 +706,7 @@ scheduledBackup:
 
 importer:
   create: false
+  replicas: 1
```

Comment:
IIRC, TiDB Lightning cannot use mulitple tikv-importer to import data right now.

Reply:
Okay，then we can use number 1 instead of `{{ .Values.importer.replicas }}`

Reply:
In fact, we can also use multiple sets of `tidb-lighting` and `tikv-importer` at the same time.